README.md

```diff
@@ -10,6 +10,24 @@ This fork uses plugins to send data to its targets.
 
 The complete [documentation](doc/vim-slime.txt) is available with `:help vim-slime`.
 
+## Upgrading from vim-slime
+
+The new vim-slime defines most of the command and mappings that legacy
+vim-slime defines, except for `<Plug>SlimeSendCell`, so your current mappings
+and configuration should still work. For cells feature, please see
+[vim-slime-cells](https://github.com/Klafyvel/vim-slime-cells/).
+
+You will need a plugin for your specific target. Please see [available target plugins](#available-plugins). 
+Please refer to the target plugin documentation and [Registering the plugin](#registering-the-plugin) 
+below for configuration.
+
+## Available plugins
+
+Here are the available target plugins. If you write a new plugin for your
+favourite target, please open a [pull request](https://github.com/jpalardy/vim-slime-ext-plugins/compare).
```

Comment:
compare? (doesn't look right)

Reply:
That's the URL the "open pull request" button points to :thinking:

Reply:
I'll replace with https://github.com/jpalardy/vim-slime-ext-plugins/edit/main/README.md

Reply:
fixed in [8fae996](https://github.com/jpalardy/vim-slime-ext-plugins/pull/6/commits/8fae996d1da07191d1e2453119863ef25d452e92)

Reply:
it's fine ... I didn't realize :+1: